lightblue-ldap-crud/src/test/java/com/redhat/lightblue/crud/ldap/translator/EntryTranslatorFromJsonTest.java

```diff
@@ -68,7 +68,7 @@ protected static Entry buildEntry(String fieldName, String metadataType, String
                 .replaceFirst("#fieldname", fieldName)
                 .replaceFirst("#value", crudValue);
 
-        EntityMetadata md = MetadataUtil.createEntityMetadata(LdapConstant.BACKEND, json(metadata), null, null);
+        EntityMetadata md = MetadataUtil.createEntityMetadata(LdapConstant.BACKEND, json(metadata, true), null, null);
```

Comment:
This is likely to be 'false', unless you want property substitution in your  metadata.